.changeset/fresh-vans-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-splunk-on-call': minor
```

Comment:
This'd be a `patch` since it's a non-breaking (arguably) fix of a bug. While on 0.x we also treat all non-breaking changes as `patch` level.